### PR TITLE
add an easy way to find yard plugins

### PR DESCRIPTION
**Pull request by noraj1337**
# Description

Add an easy way to find yard plugins.

See issue #1139

# Completed Tasks

* [x] I have read the [Contributing Guide][contrib].
* [x] The pull request is complete (implemented / written).
* [x] Git commits have been cleaned up (squash WIP / revert commits).
* [x] I wrote tests and ran `bundle exec rake` locally (if code is attached to PR).

[contrib]: https://github.com/lsegal/yard/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -669,6 +669,8 @@ To disable a plugin create the file `~/.yard/ignored_plugins` with a list
 of plugin names separated by newlines. Note that the `.yard` directory might
 not exist, so you may need to create it.
 
+You may find some useful YARD plugins on [RubyGems](https://rubygems.org/search?utf8=%E2%9C%93&query=name%3A+yard) or with a [Google avanced querry](https://www.google.com/search?q=site%3Arubygems.org+intitle%3A%22yard-%22+OR+intitle%3A%22yard_%22).
```
Comment (lsegal) [inline docs/GettingStarted.md]: typo: `querry` -> `query`

also nitpick but consider trying to enforce an 80char line width here. You can use the `[RubyGems][rubygems]` linking format with the references just below.

Comment (noraj1337) [reply docs/GettingStarted.md]: + my bad for the typo
+ I wanted to but forget about the line of 80 chars

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -669,8 +669,11 @@ To disable a plugin create the file `~/.yard/ignored_plugins` with a list
 of plugin names separated by newlines. Note that the `.yard` directory might
 not exist, so you may need to create it.
 
-You may find some useful YARD plugins on [RubyGems](https://rubygems.org/search?utf8=%E2%9C%93&query=name%3A+yard) or with a [Google avanced querry](https://www.google.com/search?q=site%3Arubygems.org+intitle%3A%22yard-%22+OR+intitle%3A%22yard_%22).
+You may find some useful YARD plugins on [RubyGems][RubyGemsQuery] or with
+a [Google avanced query][GoogleAdvancedQuery].
```
Comment (lsegal) [inline docs/GettingStarted.md]: oops, another typo: avanced -> advanced

sorry I missed this the first time!

Comment (noraj1337) [reply docs/GettingStarted.md]: My bad, so few words and so many mistakes

**Comment by lsegal** [comment]
Thanks for the doc update!

**Comment by noraj1337** [comment]
I hope you squashed that :D

**Pull request merged**